modules/machineset-yaml-aws.adoc

```diff
@@ -98,37 +115,52 @@ endif::infra[]
                   values:
                     - <infrastructure_id>-worker-sg <1>
           subnet:
+ifndef::edge[]
             filters:
               - name: tag:Name
                 values:
                   - <infrastructure_id>-private-<zone> <8>
+endif::edge[]
+ifdef::edge[]
+              id: ${localzone_subnet_id}
```

Comment:
Maybe it will be better addressed as the following to be in sync with the section `Creating a subnet in AWS Local Zones`:
```suggestion
              id: <value_of_PublicSubnetIds>
```

Reply:
Good catch. Updated!